minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -40,11 +37,13 @@
  */
 public class MesosCluster extends ExternalResource {
 
-    public static final String MINIMESOS_HOST_DIR_PROPERTY = "minimesos.host.dir";
-    public static final String MINIMESOS_FILE_PROPERTY = "minimesos.cluster";
+    private static Logger CLILOGGER = Logger.getLogger(Main.class);
```

Comment:
Please, no CLILOGGER here. Cluster can return data. Printing it out should be handled outside.
`Main` uses `Cluster`. `Cluster` should not use `Main` - cycle dependency